Sanghyun/app.js

```diff
@@ -39,6 +38,26 @@ app.get("/ping", (req, res) => {
     res.status(200).json({message : 'pong'});
 });
 
+//create users
+app.post("/users", async (req, res, next)=>{
+    const {user_name, email, profile_image, user_password} = req.body
+
+    await  appDataSource.query( 
+        `INSERT INTO users(
+          user_name ,
+          email,
+          profile_image,
+          user_password     
+        ) VALUES (?, ?, ?, ?);
+     `,
+     [user_name, email, profile_image, user_password]  
+    )        
+
+    res.status(201).json({message: "Usercreated"});
+})
```

Comment:
```suggestion
app.post("/users", async (req, res, next)=>{
    const {userName, email, profileImage, userPassword} = req.body

    await  appDataSource.query( 
        `INSERT INTO users(
          user_name ,
          email,
          profile_image,
          user_password     
        ) VALUES (?, ?, ?, ?);
     `,
     [userName, email, profileImage, userPassword]  
    )        

    return res.status(201).json({message: "Usercreated"});
})
```
javascript에서는 conding convention으로 코드 작성시 `snake_case`(밑줄로 이어주는)를 사용하는 것을 지양하고 있습니다. snake_case 대신 `camelCase`(소문자로 시작해서 대문자를 섞어주는)를 통해 작성될 수 있도록 수정해주세요.